faster_whisper/transcribe.py

```diff
@@ -82,14 +121,682 @@ class TranscriptionInfo(NamedTuple):
+            no_speech_threshold: If the no_speech probability is higher than this value AND
+                the average log probability over sampled tokens is below `log_prob_threshold`,
+                consider the segment as silent.
+            initial_prompt: Optional text string or iterable of token ids to provide as a
+                prompt for the first window.
+            prefix: Optional text to provide as a prefix for the first window.
+            suppress_blank: Suppress blank outputs at the beginning of the sampling.
+            suppress_tokens: List of token IDs to suppress. -1 will suppress a default set
+                of symbols as defined in the model config.json file.
+            enable_ta_fe: Use torch audio based kaldi fbank features for faster feature extraction.
+            prepend_punctuations: If word_timestamps is True, merge these punctuation symbols
+                with the next word
+            append_punctuations: If word_timestamps is True, merge these punctuation symbols
+                with the previous word
+            max_new_tokens: Maximum number of new tokens to generate per-chunk. If not set,
+                the maximum will be set by the default max_length.
+            clip_timestamps:
+                Comma-separated list start,end,start,end,... timestamps (in seconds) of clips to
+                process. The last end timestamp defaults to the end of the file.
+            hotwords:
+                Hotwords/hint phrases to the model. Has no effect if prefix is not None.
+            word_timestamps: Extract word-level timestamps using the cross-attention pattern
+                and dynamic time warping, and include the timestamps for each word in each segment.
+                Set as False.
+
+        Static params: (Fixed for batched version)
+            without_timestamps: Only sample text tokens, set as True.
+            max_initial_timestamp: The initial timestamp cannot be later than this, set at 0.0.
+            multilingual: If True, perform transcription on multilingual videos. Set as False.
+            output_language: Valid only if multilingual is set to True.
+                Specifies the string representing the output language. One of
+                'en' (English) or 'hybrid' (code-switched transcription). set as None.
+            condition_on_previous_text: If True, the previous output of the model is provided
+                as a prompt for the next window; disabling may make the text inconsistent across
+                windows, but the model becomes less prone to getting stuck in a failure loop,
+                such as repetition looping or timestamps going out of sync. Set as False
+            prompt_reset_on_temperature: Resets prompt if temperature is above this value.
+                Arg has effect only if condition_on_previous_text is True. Set at 0.5
+            hallucination_silence_threshold: Optional[float]
```

Comment:
Todo: support `hallucination_silence_threshold` when `word_timestamps=True`

Reply:
Added as a TODO for now.